pages/linux/lvscan.md

```diff
@@ -0,0 +1,21 @@
+# lvscan
+
+> Scan (list) all logical volumes in the system.
+> Part of the LVM (Logical Volume Manager) suite.
+> More information: <https://manned.org/lvscan>.
+
+- List all logical volumes:
+
+`lvscan`
+
+- List only active logical volumes:
+
+`lvscan --active`
+
+- List only inactive logical volumes:
+
+`lvscan --inactive`
+
+- Display logical volumes in JSON format:
+
+`lvscan --reportformat {{json}}`
```

Comment:
```suggestion
`lvscan --reportformat json`
```
JSON is mentioned explicitly in the description so no placeholder.
Consoder using "in specific format" if you want to use placeholders